.github/pr-labels.yml

```diff
@@ -0,0 +1,51 @@
+"C++":
+  - "**/*.cpp"
+  - "**/*.h"
+
+"Json":
+  - "**/*.json"
+
+"Python":
+  - "**/*.py"
+
+"Documentation":
+  - "**/*.md"
+  - doc/**/*
+
+"Code: Tooling":
+  - tools/**/*
+
+"Code: Test":
```

Comment:
```suggestion
"Code: Tests":
```

Just noticed that the existing label wasn't being used:

![labels](https://user-images.githubusercontent.com/12537966/157156377-1c2859fb-e120-401e-9cf2-148b358012c5.png)

Reply:
Ah, yeah I was planning on fixing that, also include the changes from my other fix label box pr